<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>145</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>495</width>
    <height>145</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>495</width>
    <height>145</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dodaj Produkt</string>
  </property>
  <widget class="QComboBox" name="existProducts">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>151</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="price">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Kwota:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>z≈Ç</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Produkt</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newProduktName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>151</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="AddButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zamknij</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Nowy produkt</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">errorLabel {color:red;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
